src/app/customer/Customer.tsx

```diff
@@ -121,13 +123,15 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
                         checkEmbeddedSupport={ checkEmbeddedSupport }
                         deinitialize={ deinitializeCustomer }
                         initialize={ initializeCustomer }
+                        isInitializing={ isInitializing }
                         methodIds={ checkoutButtonIds }
                         onError={ onUnhandledError }
                     />
                 }
                 defaultShouldSubscribe={ defaultShouldSubscribe }
                 email={ this.draftEmail || email }
                 isContinuingAsGuest={ isContinuingAsGuest }
+                isLoading={ isInitializing }
```

Comment:
If we go ahead with the other suggestion and collapse these 2, we can just make the `||` here:

```suggestion
                isLoading={ isContinuingAsGuest || isInitializing }
```